gb-cpu/src/microcode.rs

```diff
@@ -0,0 +1,22 @@
+pub mod controller;
+mod fetch;
+mod fetch_cb;
+pub mod opcode;
+pub mod opcode_cb;
+mod state;
+
+pub(crate) use controller::MicrocodeController;
+pub(crate) use state::State;
+
+/// Continuum allow the action to control the flow of the microcode
+pub enum Continuum {
```

Comment:
Use `ControlFlow` enum instead? (1.55 feature)

Reply:
If we move to `1.55`, I'll change that in a future PR